code/modules/mob/living/carbon/xenomorph/xenomorph_defense.dm

```diff
@@ -24,6 +24,7 @@ Contains most of the procs that are called when a xeno is attacked by something
 	amount *= 0.5 // half length
 	return ..()
 
-/mob/living/carbon/xenomorph/Paralyze(amount, updating, ignore_canstun)
-	amount *= 0.2 // replaces the old knock_down -5
+/mob/living/carbon/xenomorph/Paralyze(amount, updating, ignore_canstun, xeno_adjust_stun = TRUE)
+	if(xeno_adjust_stun)
+		amount *= 0.2 // replaces the old knock_down -5
```

Comment:
what is this

Reply:
arg added so that you can choose whether or not to apply the standard 80% reduction to stun/paralyze time for xenomorphs

Reply:
this shouldnt exist